registries/agentspace_adapter.hocon

```diff
@@ -0,0 +1,74 @@
+
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+{
+    "llm_config": {
+        "model_name": "gpt-4o",
+    },
+    "tools": [
+        # These tool definitions do not have to be in any particular order
+        # How they are linked and call each other is defined within their
+        # own specs.  This could be a graph, potentially even with cycles.
+
+        # This first guy is the "Front Man".  He is identified as such because
+        # he is the only one with no parameters in his function definition,
+        # and therefore he needs to talk to the outside world to get things rolling.
+        {
+            "name": "AgentSpace searcher",
+
+            # Note that there are no parameters defined for this guy's "function" key.
+            # This is the primary way to identify this tool as a front-man,
+            # distinguishing it from the rest of the tools.
+
+        "function": {
+            "description": "Assist caller with searching a query."
+            "parameters": {
+                "type": "object",
+                "properties": {
+                    "search_query": {
+                        "type": "string",
+                        "description": """
+    An inquiry from a user.
+    """
+                    },
+                },
+                "required": ["search_query"]
+            }
+         },
+
+            "instructions": """
+Use your tool to respond to the inquiry.
+""",
+            "tools": ["agentspace_adapter"]
+        },
+        {
+            "name": "agentspace_adapter",
+            "function": {
+                "description": """
+Assist caller with searching a query by using your tool.
```

Comment:
The description here should be the docstring telling the agent what this function does, so that it knows when and how to call it. Maybe somethings like:
```
Queries an Agentforce agent to answer the passed search query
```